app/server/controllers/login.ts

```diff
@@ -69,6 +73,23 @@ function getIdamCallback(
   };
 }
 
+function getDummyLogin(req: Request, res: Response) {
+  return res.render('dummy-login.html');
+}
+
+function postDummyLogin(getOnlineHearing) {
+  return async(req: Request, res: Response, next: NextFunction) => {
+    const email: string = req.body['username'];
+
+    try {
+      return loadHearingAndEnterService(getOnlineHearing, email, req, res);
```

Comment:
I think we might need to `await` this, as it contains promises.

Reply:
done